### DISCUSSION TITLE
OutlinedButton canot set text color

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#684] [General]
how to replace textcolor on outlinedbutton to not always blue color ?

**Comment by ndonkoHenri** [comment]
https://flet.dev/docs/controls/outlinedbutton/#style

**Comment by bobwatcherx** [comment]
https://flet.dev/docs/controls/outlinedbutton/elevatedbutton#style
i didn't find any code sample there

**Comment by ndonkoHenri** [comment]
Seems like that link is incorrect. I will correct it.
Use this example:

```python
import flet as ft

def main(page: ft.Page):
    page.add(
        ft.OutlinedButton(
            "Styled button 1",
            style=ft.ButtonStyle(
                color={
                    "hovered": ft.colors.WHITE,  # the color to be used when the button is hovered
                    "focused": ft.colors.BLUE,   # the color to be used when the button is on focus
                    "": ft.colors.YELLOW_900  # this is the color to be used by default
                }
            )
        )
    )

ft.app(target=main)
```
This is the link: https://flet.dev/docs/controls/elevatedbutton/#style but the MaterialState has an issue as per https://github.com/flet-dev/flet/issues/685, so just stick with strings("hovered", "focused", "", ...) like in my example for now :)
